### ISSUE TITLE
Could you provide train loss visualize picture by tensorboard ? And the require files not full provided.

### ISSUE THREAD

**Issue by svjack** [matenure/FastGCN#8]
I run the train_batch_multiRank_inductive_reddit_Mixlayers_sampleA.py 
The data process procedure is complete by the transferRedditDataFormat and transferRedditData2AdjNPZ in transformRedditGraph2NPZ.py which remove node without "val" in G (load by reddit.json)
in transferRedditDataFormat 
the conclusion is not  appropriate, the training loss not decrease uniformly with the training going on.
So could you provide the generated reddit.npy reddit_adj.npy 
in google drive or other server ?
Or tackle the node attribute problem ?

**Comment by matenure** [comment]
reddit_adj.npz: https://drive.google.com/open?id=174vb0Ws7Vxk_QTUtxqTgDHSQ4El4qDHt

reddit.npz: https://drive.google.com/open?id=19SphVl_Oe8SJ1r87Hr5a6znx3nJu1F2J

Thanks for the comment. I will track the attribute problem when I get some free time.

**Comment by svjack** [comment]
> reddit_adj.npz: https://drive.google.com/open?id=174vb0Ws7Vxk_QTUtxqTgDHSQ4El4qDHt
> 
> reddit.npz: https://drive.google.com/open?id=19SphVl_Oe8SJ1r87Hr5a6znx3nJu1F2J
> 
> Thanks for the comment. I will track the attribute problem when I get some free time.

Thank you to your reply.

**Comment by xz-keg** [comment]
I face the same problem. Even if I modify the preprocess program to avoid bugs, I finally get a 0.146 validation accuracy on the preprocessed data...

**Comment by thalesogoncalves** [comment]
@aviczhl2 exactly same problem here.. could you figure out what should be done to reproduce the experiment?
